test/integration/transformers.spec.ts

```diff
@@ -906,10 +909,21 @@ describe('transformers', () => {
 								},
 							},
 						},
+						input: {
+							type: 'object',
+							properties: {
+								type: {
+									const: 'card@1.0.0',
+								},
+							},
+						},
 					},
 				},
 			},
 		});
+		if (tasks.length > 1) {
+			logger.error(ctx.logContext, 'unexpected task contracts created', tasks);
```

Comment:
Instead of outputting an error about unexpected data shouldn't the test fail?

Reply:
The line below fails the test:

Reply:
Not sure it we have an standard way of adding useful "fail" messages to expect